Core/Locales/common.ptBR.lua

```diff
@@ -1,7 +1,7 @@
 local L = BigWigsAPI:NewLocale("BigWigs: Common", "ptBR")
 if not L then return end
 
--- Prototype.lua common words
+Prototype.lua palavras comuns
```

Comment:
Only things begging with `L.` are translations, what you've done here is removed a comment which is just designed to be a file header.

Same thing with the comments next to the translations, they are just there to help you understand what the translation is used for, and doesn't need translated.

The `--` must remain for those things or it will error.

Reply:
got it.

Reply:
i restored all by the enUS file.